T/TempestRemap/build_tarballs.jl

```diff
@@ -0,0 +1,91 @@
+]
+
+script = raw"""
+cd ${WORKSPACE}/srcdir/tempestremap*
+
+export CPPFLAGS="-I${includedir}"
+export LDFLAGS="-L${libdir}"
+export LDFLAGS_MAKE="${LDFLAGS}"
+CONFIGURE_OPTIONS=""
+
+autoreconf -fiv
+mkdir -p build && cd build
+
+../configure \
+  --prefix=${prefix} \
+  --host=${target} \
+  --with-blas=blastrampoline \
+  --with-lapack=blastrampoline \
+  --with-netcdf=${prefix} \
+  --with-hdf5=${prefix} \
+  --enable-shared \
+  --disable-static
+
+make LDFLAGS="${LDFLAGS_MAKE}" -j${nproc} all
+make install
+
+install_license ../LICENSE
+"""
+
+# Note: We are restricted to the platforms that NetCDF supports, the library is Unix only
+platforms = [
+    Platform("x86_64", "linux"),
+    Platform("aarch64", "linux"; libc="glibc"),
+    Platform("x86_64", "macos"),
+] 
+platforms = expand_cxxstring_abis(platforms)
+
+products = [
+    LibraryProduct("libTempestRemap", :libTempestRemap),
+    ExecutableProduct("AnalyzeMap", :AnalyzeMap_exe),
+    ExecutableProduct("ApplyOfflineMap", :ApplyOfflineMap_exe),
+    ExecutableProduct("CalculateDiffNorms",  :CalculateDiffNorms_exe),
+    ExecutableProduct("CoarsenRectilinearData", :CoarsenRectilinearData_exe),
+    ExecutableProduct("ConvertExodusToSCRIP", :ConvertExodusToSCRIP_exe),
+    ExecutableProduct("ConvertSCRIPToExodus", :ConvertSCRIPToExodus_exe),
+    ExecutableProduct("GenerateCSMesh", :GenerateCSMesh_exe),
+    ExecutableProduct("GenerateGLLMetaData", :GenerateGLLMetaData_exe),
+    ExecutableProduct("GenerateICOMesh", :GenerateICOMesh_exe),
+    ExecutableProduct("GenerateLambertConfConicMesh", :GenerateLambertConfConicMesh_exe),
+    ExecutableProduct("GenerateOfflineMap", :GenerateOfflineMap_exe),
+    ExecutableProduct("GenerateOverlapMesh", :GenerateOverlapMesh_exe),
+    ExecutableProduct("GenerateOverlapMesh_v1", :GenerateOverlapMesh_v1_exe),
+    ExecutableProduct("GenerateRLLMesh", :GenerateRLLMesh_exe),
+    ExecutableProduct("GenerateRectilinearMeshFromFile", :GenerateRectilinearMeshFromFile_exe),
+    ExecutableProduct("GenerateStereographicMesh", :GenerateStereographicMesh_exe),
+    ExecutableProduct("GenerateTestData",  :GenerateTestData_exe),
+    ExecutableProduct("GenerateTransectMesh",  :GenerateTransectMesh_exe),
+    ExecutableProduct("GenerateTransposeMap", :GenerateTransposeMap_exe),
+    ExecutableProduct("GenerateUTMMesh", :GenerateUTMMesh_exe),
+    ExecutableProduct("GenerateVolumetricMesh", :GenerateVolumetricMesh_exe),
+    ExecutableProduct("MeshToTxt", :MeshToTxt_exe),
+    ExecutableProduct("RestructureData", :RestructureData_exe),
+    ExecutableProduct("ShpToMesh", :ShpToMesh_exe),
+    ExecutableProduct("VerticalInterpolate", :VerticalInterpolate_exe),
+]
+
+dependencies = Dependency[
+    Dependency("libblastrampoline_jll"),
+    Dependency("NetCDF_jll"),
+    Dependency("HDF5_jll"),
+    # The following is adapted from NetCDF_jll
+    Dependency("LibCURL_jll"),
+    # The following libraries are dependencies of LibCURL_jll which is now a
```

Comment:
Are you sure this is still needed? I thought this had been fixed

Reply:
I can try again locally without the libcurl deps, but adding these fixed all the issues I had.

Reply:
Ok, let's go with this then.